src/Test/Perf/Utilities/RelativeDirectory.cs

```diff
@@ -9,44 +9,66 @@ namespace Roslyn.Test.Performance.Utilities
 {
     public class RelativeDirectory
     {
-        string _workingDir;
+        private string _workingDir;
 
-        public RelativeDirectory([CallerFilePath] string workingFile = "")
+
+        public RelativeDirectory()
         {
-            _workingDir = Directory.GetParent(workingFile).FullName;
         }
 
-        public string MyWorkingDirectory => _workingDir;
+        public RelativeDirectory(string workingDir)
```

Comment:
Ideally we would only be using one of the forms, but right now removing this constructor would require an awkward pattern in a few places:

```c#
var relativeDir = new RelativeDirectory();
relativeDir.SetWorkingDirectory(someKnownDirectory);
```